### DISCUSSION TITLE
Display if a content is a video

### DISCUSSION THREAD

**Discussion by Catvari** [dailydotdev/daily#918] [Feature Requests]
Hello,

It would be great to be able to see if an item is a video and not a text article because I think like me a lot of people use daily dev at office and can't watch videos.

Best regards,
Ryan

**Comment by idoshamun** [comment]
Hi @Catvari,

That's a great idea. Thanks a lot!

**Reply by idoshamun** [reply]
@Catvari happy to share it's live :)
